src/loan-options/conventional-loans/index.html

```diff
@@ -127,8 +127,8 @@ <h3 class="subhead tabbed-heading">Ask CFPB</h3>
             </a>
           </li>
           <li class="list_item">
```

Comment:
Link changes to this page require browser test updates here https://github.com/cfpb/owning-a-home/blob/master/test/browser_testing/features/loan_conventional.feature